hw/top_earlgrey/dv/chip_sim_cfg.hjson

```diff
@@ -588,9 +588,9 @@
       uvm_test_seq: chip_sw_rstmgr_alert_info_vseq
       sw_images: ["//sw/device/tests:rstmgr_alert_info_test:1"]
       en_run_modes: ["sw_test_mode_test_rom"]
-      run_opts: ["+sw_test_timeout_ns=15_000_000", "+en_scb_tl_err_chk=0",
+      run_opts: ["+sw_test_timeout_ns=15_500_000", "+en_scb_tl_err_chk=0",
```

Comment:
What's going on here? Does this test use I2C and did these changes slow it down or did something acidentally get added to the commit?

Reply:
o this is actually part of a different commit that needs to be rebased.
For whatever reason this test triggered a random seed such that alert_info started to fail.